e2e/scientists_queries.yaml

```diff
@@ -1049,7 +1049,7 @@ queries:
 
 
   - query: select_asterisk_algo-star-female-scientists
-    type: text
+    type: no-text
```

Comment:
Is that `type` actually used somewhere? I would leave `text` here,
it is a text query (anything with `contains-entity` or `contains-word` is a `text` query.
The `?t` here is a text variable. In the current code you could also write `SELECT TEXT(?t)` instead of `SELECT ?t`
for this variable, this makes no difference.